analyzers/tests/SonarAnalyzer.UnitTest/Common/RuleTest.cs

```diff
@@ -89,6 +89,7 @@ public void RulesDoNotThrow_CS()
 
             Verifier.VerifyNoExceptionThrown(@"TestCasesForRuleFailure\InvalidSyntax.cs", analyzers, CompilationErrorBehavior.Ignore);
             Verifier.VerifyNoExceptionThrown(@"TestCasesForRuleFailure\SpecialCases.cs", analyzers, CompilationErrorBehavior.Ignore);
+            Verifier.VerifyNoExceptionThrown(@"TestCasesForRuleFailure\DictionaryStressTest.cs", analyzers, CompilationErrorBehavior.Ignore);
```

Comment:
Maybe it would be nice to rename DictionaryStressTest to PerformanceTestCases. So the intention is clear and other syntaxes can be added in the future.